.github/workflows/edFi.ods.standard manual.yml

```diff
@@ -34,12 +36,34 @@ jobs:
           repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation
           path: Ed-Fi-ODS-Implementation/
           ref: main
+    - name: Is pull request branch exists in Ed-Fi-ODS-Implementation
```

Comment:
@semalaiappan How are the changes in edfi.ods.standard manual.yml related to the ticket?

Reply:
Sorry . not related to this ticket and i removed it